test/integration/scheduler/scheduler_test.go

```diff
@@ -1229,32 +1232,39 @@ var _ = ginkgo.Describe("Scheduler", func() {
 				},
 			}
 			gomega.Expect(k8sClient.Create(ctx, matchingNS)).To(gomega.Succeed())
+
+			strictFIFOLocalQueue = testing.MakeLocalQueue("strict-fifo-q", matchingNS.Name).ClusterQueue(strictFIFOClusterQ.Name).Obj()
+			gomega.Expect(k8sClient.Create(ctx, strictFIFOLocalQueue)).Should(gomega.Succeed())
+
+			cqTeamA = testing.MakeClusterQueue("team-a").ResourceGroup(*testing.MakeFlavorQuotas(onDemandFlavor.Name).Resource(corev1.ResourceCPU, "5").Obj()).Cohort(chName).Obj()
```

Comment:
for readability, split this into multiple lines, similar to how we do for `strictFIFOClusterQ`.